source/api/commands/server.md

```diff
@@ -53,7 +51,7 @@ Option | Default | Description
 Option | Default | Description
 --- | --- | ---
 `enable` | `true` | pass `false` to disable existing route stubs
-`force404` | `false` | forcibly send XHR's a 404 status when the XHR's do not match any existing
+`force404` | `false` | forcibly send XHR's a 404 status when the XHR's do not match any existing route
```

Comment:
I thought the sentence ended abruptly so I added a trailing _route_ to make it read smoother.